website/static/js/myProjects.js

```diff
@@ -121,7 +121,7 @@ function NodeFetcher(type, link, handleOrphans, regType, regLink) {
         link = link ? link : $osf.apiV2Url('users/me/nodes/', { query : { 'filter[preprint]': true, 'related_counts' : 'children', 'embed' : ['contributors', 'preprints'], 'fields[users]' : sparseUserFields, 'fields[contributors]' : sparseContributorFields}});
     }
 
-    this.nextLink = link ? 
+    this.nextLink = link ?
         link + '&version=2.2' :
```

Comment:
It wouldn't be a bad idea to upgrade to 2.6 for this, but I don't think it would hold anything up leaving it at 2.2.